### PR TITLE
Add cross-language efforts to roadmap index page

### PR DESCRIPTION
**Pull request by tweise**
Though currently also visible from left nav, this makes it more discoverable for users.

### CODE REVIEW

**Comment by tweise** [comment]
R: @chadrik

File: website/src/roadmap/index.md [hunk 1]
```diff
@@ -34,16 +34,18 @@ Below are some highlights for the project as a whole.
 
 Portability is the primary Beam vision: running pipelines authors with _any SDK_
```
Comment (chadrik) [inline website/src/roadmap/index.md]:
```suggestion
 Portability is the primary Beam vision: running pipelines authored with _any SDK_
```

File: website/src/roadmap/index.md [hunk 2]
```diff
@@ -34,16 +34,18 @@ Below are some highlights for the project as a whole.
...
 See the details on the [Portability Roadmap]({{site.baseurl}}/roadmap/portability/)
 
-## Python on Flink
+## Cross-language transforms
 
-A major highlight of the portability effort is the effort in running Python pipelines
-the Flink runner.
-
-You can [follow the instructions to try it out]({{site.baseurl}}/roadmap/portability/#python-on-flink)
+As benefit of the portability effort, we are able to utilize Beam transforms across SDKs.
```
Comment (chadrik) [inline website/src/roadmap/index.md]:
```suggestion
As a benefit of the portability effort, we are able to utilize Beam transforms across SDKs.
```

**Comment by chamikaramj** [comment]
LGTM. Thanks!

**Pull request merged**